logconfiguration.tf

```diff
@@ -0,0 +1,41 @@
+# define various task definition log configuration option blocks
+# and use a module var to switch between different log drivers
+locals {
+  all_log_configurations = {
+    awslogs = {
+      logDriver = "awslogs",
+      options = {
+        awslogs-group         = join("", aws_cloudwatch_log_group.keycloak-log-group.*.id),
+        awslogs-region        = var.aws_region,
+        awslogs-stream-prefix = "log"
+      }
+    },
+    splunk = {
+      logDriver = "splunk",
+      options = {
+        splunk-url    = var.splunk_http_endpoint,
+        splunk-index  = "keycloak-${var.environment}"
+        splunk-format = "raw"
+      }
+      secretOptions = [
+        {
+          name      = "splunk-token",
+          valueFrom = join("", aws_secretsmanager_secret.keycloak-splunk-token.*.arn)
```

Comment:
suggestion(non-blocking): could you make the token secret ARN also an optional configuration? That would allow you to pass in an existing secret to share between the different Keycloak environments.